arbitrum-docs/how-arbitrum-works/07-interactive-fraud-proofs.mdx

```diff
@@ -15,53 +17,50 @@ RBlocks 93 and 95 are siblings (they both have 92 as predecessor). Alice is bond
 
 At this point we know that Alice and Bob disagree about the correctness of <a data-quicklook-from="rblock">RBlock</a> 93, with Alice committed to 93 being correct and Bob committed to 93 being incorrect. (Bob is bonded on 95, and 95 implicitly claims that 92 is the last correct RBlock before it, which implies that 93 must be incorrect.)
 
-Whenever two bonders are bonded on sibling RBlocks, and neither of those bonders is already in a <a data-quicklook-from="challenge">Challenge</a>, anyone can start a challenge between the two. The rollup protocol will record the challenge and referee it, eventually declaring a winner and confiscating the loser’s bond. The loser will be removed as a bonder.
+Whenever two (or more) bonders are bonded on sibling RBlocks, a challenge begins between all of them. (There can only be one bonder per RBlock, but multiple RBlocks may participate in a challenge.) The rollup protocol will record the challenge and referee it, eventually declaring a winner and confiscating the losers' bonds. This will cause the losers to be removed as bonders.
```

Comment:
```suggestion
Whenever two (or more) bonders are bonded on sibling RBlocks, a challenge begins between all of them. (There can only be one bonder per RBlock, but multiple RBlocks may participate in a challenge.) The Rollup protocol will record and referee the challenge, eventually declaring a winner and confiscating the losers' bonds. This will cause the losers to be removed as bonders.
```